tfx/dsl/component/experimental/executor_specs.py

```diff
@@ -90,3 +90,12 @@ def __init__(
     self.image = image
     self.command = command
     super(TemplatedExecutorContainerSpec, self).__init__()
+
+  def __eq__(self, other):
```

Comment:
```suggestion
  def __eq__(self, other) -> bool:
```
Perhaps it's preferable to type-hint the return value when it's possible